docker/docker-compose.yml

```diff
@@ -1,24 +1,8 @@
 version: '3.8'
 services:
-  vocascan:
-    image: vocascan/vocascan-server
-    restart: always
-    tty: true
-    depends_on:
-      - db
-    environment:
-      PORT: '8000'
-      DB_DIALECT: 'postgres'
-      DB_HOST: 'db'
-      DB_PORT: '5432'
-      DB_USERNAME: 'vocascan'
-      DB_PASSWORD: 'vocascan'
-      DB_DATABASE: 'vocascan'
-      SALT_ROUNDS: '10'
-      JWT_SECRET: ''
-    ports:
-      - '8000:8000'
```

Comment:
why did you removed that service?

Reply:
I accidentally pushed my testing docker-compose file and didn't notice it. I already fixed that.

Reply:
I accidentally pushed my testing docker-compose file and didn't notice it. I already fixed that.